daemon/daemon_no_aufs.go

```diff
@@ -1,4 +1,4 @@
-// +build exclude_graphdriver_aufs,linux
+// +build exclude_graphdriver_aufs,linux !linux
```

Comment:
@jhowardmsft This adds a new file which makes this no_aufs source build on windows.